uaclient/entitlements/tests/test_fips.py

```diff
@@ -352,7 +352,7 @@ def test_non_enabled_passed_through(
             ),
             (
                 {"kernel": "4.4.0-148-generic"},
-                status.ApplicationStatus.PENDING,
+                status.ApplicationStatus.ENABLED,
```

Comment:
The test this change is attached to still has "pending" in its name. Can we confirm that it's still a useful test, and fix the name if it is?

Reply:
Test is still useful as we do surface unique application status messages based on the running kernel. Even though ua status cli doesn't distinguish between ENABLED, 'still needs reboot' and ENABLED, 'active'.